common-npm-packages/artifacts-common/module.json

```diff
@@ -10,6 +10,8 @@
         "CredProvider_InstallingNetCoreTo_NuGetAuthenticateV0": "Installing the Azure Artifacts Credential Provider (.NET Core) to '%s'. This credential provider is compatible with dotnet SDK 2.1 or later.",
         "CredProvider_InstallingNetCoreTo_NuGetAuthenticateV1": "Installing the Azure Artifacts Credential Provider (.NET Core) to '%s'. This credential provider is compatible with .NET SDK 6 or later.",
         "CredProvider_InstallingNetFxTo": "Installing the Azure Artifacts Credential Provider (.NET Framework) to '%s'. This credential provider is compatible with nuget.exe 4.8.0.5385 or later, and MSBuild 15.8.166.59604 or later.",
+        "CredProvider_Error_MultipleServiceConnections": "Multiple Service connections are not supported. Only one service connection should be used per task execution.",
+        "CredProvider_Error_ServiceConnectionExists" : "An existing Service connection already exists for this feed.",
```

Comment:
For the Tasks repo, we learned from the last task updates that we need to include the en-us string to allow the localization pipeline to update the other localization.

Reply:
done